app/views/manage_user/index.html.erb

```diff
@@ -2,6 +2,8 @@
 	<body class='index_body' id='index_body'>
 		<div class='shade pt-5'>
 			<div class="accordion" id="accordionExample">
+			<%= link_to 'define role', permissions_path, :class=>'btn btn-danger btn-sm' %>
```

Comment:
Why danger class? Dont want red button. Change to some other class

Reply:
removed the link because I replaced the controller from the new controller to the existing one.